### PR TITLE
Create Book model

### PR DESCRIPTION
**Pull request by alexdavcudriz**
## Summary
- Creation of the book model
## Trello Card
https://trello.com/c/4e9JmQGU/14-crear-modelo-de-book

### CODE REVIEW

File: src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    public Book() {
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (jdhiguita) [inline src/main/java/wolox/training/models/Book.java]: In Postgres when GenerationType.AUTO is used, it generates an incremental IDs shared between the schema tables. This happens only for this database engine. To generate independent IDs for each table use GenerationType.SEQUENCE.

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    private long id;
+    @Column(nullable = false)
+    private String Genre;
```
Comment (jdhiguita) [inline src/main/java/wolox/training/models/Book.java]: The @column annotations are required to be able to generate the model in the database.
In this case I would add the @column (nulleable = false) annotation in conjunction with the @NotNull annotation(this should in the rest of the fields except for the id).

File: src/main/java/wolox/training/models/Book.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    @Column(nullable = false)
+    private String Subtitle;
+    @Column(nullable = false)
+    private String Publisher;
+    @Column(nullable = false)
+    private String Year;
+    @Column(nullable = false)
+    private int Pages;
+    @Column(nullable = false)
+    private String isbn;
```
Comment (jdhiguita) [inline src/main/java/wolox/training/models/Book.java:45]: In these cases where the fields need to be unique ,I would not just add @column (nullable = false)
I would also add the following:
@column (nullable = false, unique = true)

**Review by jdhiguita — COMMENTED** [review]
Good job! Some comments

**Review by jdhiguita — APPROVED** [review]

**Review by facucompe — APPROVED** [review]

**Pull request merged**